styles.css

```diff
@@ -5,439 +5,1094 @@
+    padding: 15px;
+    border: none;
+    margin-top: 385px;
+    width: 100%;
+    font-weight: 700;
+    font-size: 17px;
+    display: block;
+  }
+
+  .post11 {
+    display: none;
+  }
+
+  .post2 {
+    background-image: url("images/card2.png");
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  .second-title h1 {
+    width: 337px;
+    height: 81px;
+    color: #fff;
+    font-size: 32px;
+    font-weight: 400;
+    line-height: 44px;
+    margin: 10px;
+  }
+
+  .second-title {
+    margin: 180px 0 8px 13px;
+  }
+
+  .post-title h1 {
+    display: none;
+  }
+
+  .post-content {
+    color: #fff;
+    font-family: 'inter', sans-serif;
+    font-weight: 400;
+    font-size: 17px;
+    margin: 10px 0 15px 20px;
+  }
+
+  .btn-courses button {
+    font-family: 'Inter', sans-serif;
+    height: 40px;
+    font-size: 15px;
+    padding: 8px;
+    background-color: rgba(131, 130, 130, 0.5);
+    color: #fff;
+    margin-left: 20px;
+    border: none;
+  }
+
+  .button2 {
+    display: none;
+  }
+
+  .post3 {
+    background-image: url("images/card3.png");
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  .post4 {
+    background-image: url("images/back2.png");
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  .post5 {
+    background-image: url("images/card2.png");
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  .post6 {
+    background-image: url("images/card3.png");
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  .card {
+    margin-top: 20px;
+    display: grid;
+    grid-template-columns: 369px 369px 369px;
+    grid-gap: 25px;
+    justify-content: center;
+  }
+
+  .about {
+    margin-top: 200px;
+    background: #1c1a19;
+    padding-bottom: 200px;
+    background-image: url('images/illustration-about\ me@2x.png');
+    background-repeat: no-repeat;
+    background-size: 30%;
+    background-position: right 70px;
+  }
+
+  .about-content {
+    margin-left: 70px;
+    padding-top: 100px;
+  }
+
+  .about-title {
+    color: #ff6b00;
+    font-family: 'Crete Round', serif;
+    font-weight: 400;
+    font-size: 72px;
+    line-height: 81.5px;
+    width: 566px;
+    height: 78px;
+  }
+
+  .about-description {
+    margin-top: 30px;
+    font-family: 'Inter', sans-serif;
+    font-weight: 400;
+    font-size: 20px;
+    color: #fff;
+    line-height: 32px;
+    width: 566px;
+    height: 148px;
+  }
+
+  .button {
+    margin-top: 30px;
+    width: 161px;
+    height: 48px;
+  }
+
+  .project-exp {
+    background: #1c1a19;
+  }
+
+  .line {
+    margin-top: 120px;
+    border: 1px solid #fff;
+    width: 1156px;
+    margin-left: 60px;
+  }
+
+  ul {
+    color: #fff;
+    list-style-type: none;
+  }
+
+  .exp {
+    display: grid;
+    grid-template-columns: 320px 320px 320px;
+    grid-column-gap: 120px;
+    margin-top: 100px;
+    margin-left: 100px;
+  }
+
+  .experience {
+    display: flex;
+  }
+
```

Comment:
- Please, let's align the experience section in the center to be close to the Figma design below:
- Let's use the correct font-family in the list items as you've used another font that is not the same as the one used. See the image below:

*your design*
![Screenshot from 2023-01-27 12-50-00](https://user-images.githubusercontent.com/59397159/215080363-36e1608e-ab74-4b83-b598-66f86758fa06.png)


*Figma design*
![Screenshot from 2023-01-27 12-49-08](https://user-images.githubusercontent.com/59397159/215080371-78267773-390c-4a8e-99c1-d856acc420a1.png)